modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourcePerformanceTest.java

```diff
@@ -231,48 +233,20 @@ public void testMultipleThreadsAddUsers() throws Exception {
 			for (Future<?> future : futures) {
 				future.get();
 			}
-
-			long endTime = System.currentTimeMillis() - startTime;
-
-			if (_log.isInfoEnabled()) {
-				_log.info(
-					StringBundler.concat(
-						_threadsCount, " threads each added ",
-						usersCountPerThread, " users in ", endTime, " ms"));
-
-				double tps =
-					(double)usersCountPerThread * _threadsCount / endTime *
-						1000;
-
-				_log.info("TPS: " + String.format("%.2f", tps));
-			}
 		}
 	}
 
 	@Test
 	public void testSingleThreadAddUsers() {
 		List<String> jsons = _createJSONs(_usersCount);
 
-		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
```

Comment:
@slnn Why removing the `LoggerTestUtil` usage when you add `PerformanceTimer` usage? Please don't just remove it, if it's no longer needed, remove it in a separate commit.